tests/src/test/scala/org/apache/openwhisk/core/loadBalancer/test/ShardingContainerPoolBalancerTests.scala

```diff
@@ -96,7 +96,7 @@ class ShardingContainerPoolBalancerTests
     IndexedSeq.fill(count)(new NestedSemaphore[FullyQualifiedEntityName](max))
 
   def lbConfig(blackboxFraction: Double) =
-    ShardingContainerPoolBalancerConfig(blackboxFraction, 1)
+    ShardingContainerPoolBalancerConfig(1.0 - blackboxFraction, blackboxFraction, 1)
```

Comment:
Do we need a test to assert that we correctly handle overlapping pools? I think adding a test setting both values to 1 (which should then yield the same pools) should be sufficient.